visma/gui/window.py

```diff
@@ -77,6 +80,17 @@ def popupBrowser(self):
         web.show()
         w.show()
 
+    def addEquations(self):
+        options = QFileDialog.Options()
+        options |= QFileDialog.DontUseNativeDialog
+        fileName, _ = QFileDialog.getOpenFileName(self, "Add Custom Equations", "", "All Files (*);;Visma Files (*.vis)", options=options)
+        tempList = []
+        with open(fileName) as fileobj:
+            for line in fileobj:
+                for ch in line:
+                    tempList.append(ch)
+        self.workSpace.textedit.append("".join(tempList))
```

Comment:
Instead of adding the equations to the textbox, can you load these equations to the list in equation history? For doing this you will have to append the equations from the file to `self.equation` list and then to update the equations list widget use `self.addEquation()`. These two can be found in the `WorkSpace(QWidget)` class. Also can you rename `addEquations` to `loadEquations` since `addEquation` function already exists.

Reply:
If I understand you correctly,  you want equations to be loaded from file to history first and then from history to textbox? 
Instead of directly loading them to the textbox.  Right?

Reply:
> If I understand you correctly, you want equations to be loaded from file to history first 

Yes.

> and then from history to textbox?

No need to load them from history to textbox programmatically. It's up to the user to use the equations from history.

Loading them into the textbox might not be appropriate as there can be a large number of equations.